console-subscriber/tests/support/mod.rs

```diff
@@ -1,14 +1,14 @@
-use futures::Future;
+use std::{future::Future, task::Poll};
+
+use tokio::task::JoinHandle;
```

Comment:
Maybe this change is unnecessary. But I guess it is OK.

Reply:
The previous one was probably incorrect, we should be using `Future` from the std library.